packages/web-components/fast-foundation/src/tabs/tabs.ts

```diff
@@ -162,7 +171,11 @@ export class Tabs extends FoundationElement {
     }
 
     private setTabs = (): void => {
-        const gridProperty: string = this.isHorizontal() ? "gridColumn" : "gridRow";
+        const gridHorizontalProperty: string = "grid-column";
+        const gridVerticalProperty: string = "grid-row";
```

Comment:
Shouldn't these be "gridColumn" and "gridRow" since they are being used on the JavaScript style object?

Reply:
this is an interesting question, this does work, but maybe it would be better to use it the way you suggest. I will test it out and change to that if it works the same

Reply:
it did work so I Updated it, thank Rob